src/regtech_cleanup_api/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+from urllib import parse
+from typing import Any
+
+from pydantic import field_validator, ValidationInfo
+from pydantic.networks import PostgresDsn
+from pydantic_settings import BaseSettings
+
+from regtech_api_commons.oauth2.config import KeycloakSettings
+from regtech_regex.regex_config import RegexConfigs
+
+env_files_to_load = [".env"]
+if os.getenv("ENV", "LOCAL") == "LOCAL":
+    file_dir = os.path.dirname(os.path.realpath(__file__))
+    env_files_to_load.append(f"{file_dir}/../.env.local")
+
+
+class Settings(BaseSettings):
+    db_schema: str = "public"
+    db_name: str
+    db_user: str
+    db_pwd: str
+    db_host: str
+    db_scheme: str = "postgresql+asyncpg"
+    db_logging: bool = False
+    conn: PostgresDsn | None = None
+
```

Comment:
One thing to keep in mind is we will need to access to both the user-fi database and the filing database, different schemas/users/pwds/etc.  

Also, I think we want to avoid asyncio stuff.  Le is in the process of evaluating all that.  I think for the db connections it's probably ok, but asyncio can cause a trickle down effect where everything ends up needing to be async and that causes issues.  So see if we can approach this without any asyncio first.

Reply:
Noted!